spint/flow_accessibility.py

```diff
@@ -0,0 +1,138 @@
+# coding=utf-8
+# 3. FlowAccessibility = Accessibility of flow taking existing destinations
+import numpy as np
+import pandas as pd
+import itertools
+from timeit import default_timer as timer
+
+def generate_dummy_flows():
```

Comment:
If you prefix this with a `_`, like `_generate_dummy_flows()`, then python will "hide" it from users and avoid showing it when people look at the module. 

You can see this, for example, in [esda/map_comparison.py](https://github.com/pysal/esda/blob/master/esda/map_comparison.py#L13) where the `_overlay` and `_cast` functions are used by the module, but we don't want the users to see them/use them directly.